pint.json

```diff
@@ -6,6 +6,10 @@
         "new_with_braces": {
             "anonymous_class": false,
             "named_class": false
-        }
+        },
+        "ordered_imports": {
+            "sort_algorithm": "length"
+        },
+        "no_unused_imports":true
     }
```

Comment:
These two rules ensure that the imports are ordered based on their length and unused imports will be removed automatically when you run the linter